test/unit/test_util.c

```diff
@@ -113,6 +113,42 @@ bool test_autonames(void) {
 	mu_end;
 }
 
+bool test_file_slurp(void) {
+
+#ifdef __WINDOWS__
+#define S_IRWXU _S_IREAD | _S_IWRITE
+#endif
+
+	char* test_file =  "./empty_file";
+	size_t s;
+	char* content;
+	char* some_words = "some words";
+
+	close(open(test_file, O_CREAT, S_IRWXU));
```

Comment:
missing space before (. also may be worth checking the return value of open to avoid calling close(-1) with an assert

Reply:
done, even if I am unsure on how we want to react exactly if open fail. I used some assert on the value after a quick reading of others tests cases, but I will dig further.